app/views/tasklist/by_title.erb

```diff
@@ -0,0 +1,30 @@
+<h1> class = "Bytitle">Task List</h1>
+<%params%>
+<section class="section">
+
+
+<ul class = "by_title">
+  <%@tasklist.each do |task|%>
+<li>
```

Comment:
Here again and in other places within your html, more indenting would be a great help.